docs/91_version_history.rst

```diff
@@ -33,175 +33,111 @@ Release Date: 2025-03-13
        - Documentation has been added to the plugin developers section of the documentation that describes how to persist data between restart of services or the entire system.
-       - 
-       - 
        - In an attempt to make it easier to find plugins, new subsections have been added to the documentation section that lists the available plugins.  It now includes a number of sections that categorise the plugins into those with related functionality. 
-       - 
        - The documentation section for plugin developers writers has been updated with the latest mechanisms to run Fledge services under memory analysis tool, valgrind.
-       - A new section has been added to the tuning section of the documentation that discusses tuning the purge processes within Fledge.
        - The Fledge documentation has been updated to reflect the new security defaults in the 3.0 release and to include a lengthier discussion of other optional security features.
-       - 
-       - 
-       - 
-       - 
-       - 
-       - 
-       - Monitoring of the ingest rate has now been added to the south service. If the ingest rate is seen to suddenly fall an alert will be raised to the user. If the rate returns to previous levels then the alert will be cleared. This monitoring may not be suitable for services using asynchronous south plugins or that only forward changes of values and can be disabled in the advanced configuration options of the south service.
+       - Updates to the documentation have been added that describe how a Fledge instance can be connected to a PostgreSQL instance hosted on a different host or container.
+       - The plugin developers guide has been updated to include more information on the 3.0.0 version of the south plugin interface.
 
 
     - Bug Fix:
 
-       - 
        - An issue with macro substitution incorrectly handling default values has been resolved.
-       - 
-       - 
-       - 
-       - 
-       - 
-       - An issue that could cause persisted data from plugins not to be written on the second and subsequent restarts of a service has been resolved.
-       - 
-       - 
-       - 
-       - 
-       - 
+       - An issue that could cause a problem if certain characters were used in the asset names has been resolved. 
+       - An issue with mixed case user names mismatching names in authentication certificates has been resolved. This prevented users with mixed case names correctly authenticating   using certificates. 
+       - A number of issues in the handling of plugin configuration updates have been addressed. This allows for plugin configuration to be updated in new versions of the plugins while migrating the older configuration to the new configuration. In particular this aids the transition of configuration items previously entered as a JSON structure to the new list style of configuration item.
```

Comment:
We should include the plugin names as well, or list them under the 'Plugin' section.